src/test/java/org/fundacionjala/coding/maria/Kata5Test.java

```diff
@@ -0,0 +1,78 @@
+package org.fundacionjala.coding.maria;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test to kata5.
+ */
+public class Kata5Test {
+    /**
+     * Test testComplementaryDNA.
+     */
+    @Test
+    public void testComplementaryDNA() {
+        Kata5 dna = new Kata5();
+        assertEquals("TAACG", dna.makeComplement("ATTGC"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "TAACG" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "ATTGC" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)